### ISSUE TITLE
The docs do not describe basic commands

### ISSUE THREAD

**Issue by skorokithakis** [jdx/mise#2391]
The docs are frustrating because they don't mention basic commands like install, etc. They go through: 
* Installation/uninstallation (fine)
* Project info (I don't really care right now, I want to use it)
* Coming from rtx (I didn't come from rtx)
* Configuration (why would I want to configure it when I barely know what it does?)
* CI (I definitely don't want it on CI before first use)
* Demo (this has two commands, but in a frustrating unseekable video format)

After installation, I want to know:

* How do I install something globally?
* How do I add something to a project?
* How do I create a `.mise.toml` from the cli so I can keep the versions I installed?
* How can I install the `latest` of something but have its version persisted in the `.mise.toml`?

Please write something up that addresses those use cases, something like "Using Mise". I'd do it myself but I don't know enough, because I always struggle to find the commands.

**Comment by roele** [comment]
Valid point.

The "Getting Started" section is mostly about installation. I would probably try to keep it minimal, explaining the basic first steps to use `mise` addressing most of the questions you mention. The Demo video could also be part of "Getting Started" but is mostly a visual gimmick.

The whole "(Alternative) Installation" section should be split out into its own page which can be referred to from "Getting Started".

**Comment by skorokithakis** [comment]
I agree, that would be a good way to structure it. From a "getting started" page I expect to see:

* The most common installation method (I guess `curl | bash`, these days), with a link to "other installation options" (which can be to an "Installation" page that can include uninstallation).
* Very common use cases (use global, use local, searching for packages, installing uninstalling, basic `mise.toml` usage, with links to more advanced topics.

Basically, if I haven't used a software for more than a few days, I expect the "getting started" page to be my quick references for the overwhelmingly most common actions. I'm OK to go hunt in the config page for something obscure, but there should be a page with the everyday usage stuff.

**Comment by jdx** [comment]
@roele if you're up for it, it might be good to walk through the docs together over zoom or something since there's a ton of complications that probably won't seem obvious.

For example, one struggle I've had is whether the docs should use `~/.local/bin/mise` or just `mise`. Depending on how you install the CLI one of those is probably correct and the other won't work. I believe this is explained, but it's a gotcha people run into.

**Comment by ralgozino** [comment]
FWIW, just to give a different data point: I've found everything I needed, and more, in the docs.

I think that the main difference is that I come from `asdf` and `rtx`, so I already knew the main concepts and the migration docs where super useful and valuable to me.

It may be that now that the tool has matured and you have new comers that are not migrating from asdf (or rtx) the docs need to be restructured a little.

I'd like to take the chance to thank @jdx for a great tool (life improvement over 9000 from `asdf`) and dedicating a lot of effort to the docs, having personally written docs for some projects I know that it is not an easy task, and having docs in an opensource project it not to take for granted.

**Comment by jadejr** [comment]
I didn't come from rtx and i definitely had trouble grasping what to do. I had some basic ide aof what I could do, since I was already using tools like volta and direnv.  I think what I needed was more of a "cookbook" set of examples for doing common things using a subset of popular plugins. I may not use those plugins, but I could infer a lot from context.

**Comment by jdx** [comment]
the cli reference should have examples for every command: https://mise.jdx.dev/cli/
